### PR TITLE
python-spyder: dependencies update

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

**Comment by Biswa96** [comment]
In my opinion, a patch file with all those modfications would be better.

**Comment by raedrizqie** [comment]
> In my opinion, a patch file with all those modfications would be better.

ok.. i will update it

File: mingw-w64-python-spyder/spyder-requirements-fix.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
+--- a/setup.py	2021-09-16 09:18:30 +0800
++++ b/setup.py	2021-11-20 17:41:22 +0800
+@@ -211,7 +211,7 @@
+     'ipython>=7.6.0',
+     'jedi>=0.17.2,<0.19.0',
+     'jsonschema>=3.2.0',
+-    'keyring>=17.0.0',
++    'keyring',
```
Comment (lazka) [inline mingw-w64-python-spyder/spyder-requirements-fix.patch:8]: I hope fixed here: 283d66bb008c5dc73a9b

File: mingw-w64-python-spyder/spyder-requirements-fix.patch [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+-    'pyqt5<5.13',
+-    'pyqtwebengine<5.13',
++    'pyqt5>=5.13',
++    #'pyqtwebengine<5.13',
+     'python-lsp-server[all]>=1.2.2,<1.3.0',
+     'pyxdg>=0.26;platform_system=="Linux"',
+     'pyzmq>=17',
+     'qdarkstyle==3.0.2',
+-    'qstylizer>=0.1.10',
++    'qstylizer',
```
Comment (lazka) [inline mingw-w64-python-spyder/spyder-requirements-fix.patch:29]: fixed in #10135

File: mingw-w64-python-spyder/spyder-requirements-fix.patch (hunk 1 above)
Comment (raedrizqie) [reply mingw-w64-python-spyder/spyder-requirements-fix.patch:8]: i think i tried before.. using setuptools-scm.. but still failed.. hopefully your fix will work.

maybe we have to remove the `.github` dir first

File: mingw-w64-python-spyder/spyder-requirements-fix.patch [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+     'qtawesome>=1.0.2',
+     'qtconsole>=5.1.0',
+     'qtpy>=1.5.0',
+-    'rtree>=0.9.7',
++    #'rtree>=0.9.7',
```
Comment (lazka) [inline mingw-w64-python-spyder/spyder-requirements-fix.patch:34]: adder here: #10138

File: mingw-w64-python-spyder/spyder-requirements-fix.patch (hunk 1 above)
Comment (lazka) [reply mingw-w64-python-spyder/spyder-requirements-fix.patch:8]: hm, ok, I'll check

Comment (lazka) [reply mingw-w64-python-spyder/spyder-requirements-fix.patch:8]: yes, it's fixed now

**Pull request merged**